Gemfile

```diff
@@ -36,6 +36,7 @@ gem "feedjira", "~> 3.2" # A feed fetching and parsing library
 gem "field_test", "~> 0.4" # A/B testing
 gem "flipper", "~> 0.22.1" # Feature flipping / flags for Ruby
 gem "flipper-active_record", "~> 0.22.1" # Store Flipper flags in ActiveRecord
+gem "flipper-active_support_cache_store", "~> 0.22.1" # Cache feature flags for a short time
```

Comment:
I originally rewrote the `FeatureFlag` module to do all this but then discovered that _of course_ there's an official Flipper plugin for this.